### PR TITLE
tasks/cephfs: test 'df' output in volumeclient

### PR DESCRIPTION
**Pull request by jcsp**
Fixes: http://tracker.ceph.com/issues/15599
Signed-off-by: John Spray <john.spray@redhat.com>

### CODE REVIEW

**Comment by jcsp** [comment]
Companion to https://github.com/ceph/ceph/pull/8747

File: tasks/cephfs/test_volume_client.py [hunk 1]
```diff
@@ -125,11 +125,39 @@ def test_lifecycle(self):
...
+
+        # global: df should see all the writes.  This is a > rather than a
+        # == because the global spaced used includes all pools
+        self.assertGreater(self.mount_a.df()['used'],
+                           (data_bin_mb + other_bin_mb) * 1024 * 1024)
+
+        # Hack: do a metadata IO to kick rstats
+        self.mounts[2].run_shell(["touch", "foo"])
+
+        # volume: df should see the one megabyte consumed from quota, same
```
Comment (ajarr) [inline tasks/cephfs/test_volume_client.py]: four megabytes?

Comment (jcsp) [inline tasks/cephfs/test_volume_client.py]: Done.

**Comment by gregsfortytwo** [comment]
@ajarr, is this good?

**Comment by ajarr** [comment]
@gregsfortytwo, lgtm.

**Pull request merged**